src/foam/u2/crunch/CapabilityStore.js

```diff
@@ -247,6 +247,19 @@ foam.CLASS({
         });
       });
     },
+    
+    /**
+     * Fetch capabilities intended to be rendered on the view.
+     */
+    async function getCapabilities() {
```

Comment:
Believe we should move this to crunchService so that it can be used outside this view, we should also reevaluate the need for CapabilityCategory and VisibleCapabilityDAO

Reply:
this function does not really belong in the crunchservice as it is grabbing the capabilities from the themeDomainDAO which is independent. featuredCapabilities, however, is calling the crunch service directly

Reply:
sarthak's idea is to unify the functionality of capabilityCategory / visibleCapabilityDAO and the themeDomainDAO query into a single property rather than having a function. we also have to make the outcome of the call predictable, because this function will not produce a predictable result.

basically put:
* convert this into a prop / factory
* kill no longer needed props